### DISCUSSION TITLE
iphone tracking accuracy issue

### DISCUSSION THREAD

**Discussion by shapley** [opengisch/QField#5044] [Q&A]
Hi,

One of our QField users on iphone is experiencing some 'tracking' accuracy issues. Attached is a screenshot from his device the tracking line is in 'green' with the (actual) route in 'yellow'. What might explain this? I'm not sure what model iphone it is.

![iphone_accuracy_issue](https://github.com/opengisch/QField/assets/7022885/c1f3b844-16d8-4947-a665-d85b56bde2d6)

Many thanks

**Comment by SeqLaz** [comment]
Hello @shapley, you can improve the location accuracy by activating the Accuracy indicator and the Accuracy requirement in QField.
![image](https://github.com/opengisch/QField/assets/77129293/e9a95b01-8484-4006-afa1-9d141200dc54)

Also, for tracking, it is a good practice to set constraints on the track by specifying the minimum time and distance required for each vertice. For more details, please check the official documentation at this link: https://docs.qfield.org/how-to/tracking/
